src/App.tsx

```diff
@@ -174,7 +173,6 @@ export const AppContent = ({
                                     restVirksomhetsdata={restvirksomhetsdata}
                                 />
                             </Sammenligningspanel>
-                            <KalkulatorPanel liten />
                             <Historikkpanel />
```

Comment:
<img width="800" alt="fjerne-innganger-med-2-paneler" src="https://user-images.githubusercontent.com/322962/174074429-bcda8a08-0b7e-4ac0-a331-8f62468fae0c.png">
<img width="788" alt="master-med-4-paneler" src="https://user-images.githubusercontent.com/322962/174074441-0ae85953-597f-4e09-a087-5fa03bbf1f01.png">
Det mangler når det bare er to panel som skal vises. 
"Kurskalender" vises bare når det er tilgjengelig kurs tror jeg. 
Det vil si at vi kan nå ha bare to paneler å vise frem (Historikk og Samtalestøtte-podlet). Mulig dette kunne hatt skedd før men vi gikk aldri ned til to paneler da vi kunne vise Kalkulator i tillegg.

Reply:
Fiksa!